lib/rubocop/cop/lint/empty_interpolation.rb

```diff
@@ -19,12 +19,23 @@ class EmptyInterpolation < Base
         MSG = 'Empty interpolation detected.'
 
         def on_interpolation(begin_node)
+          return if in_percent_w_upper_literal?(begin_node)
+
           node_children = begin_node.children.dup
           node_children.delete_if { |e| e.nil_type? || (e.basic_literal? && e.str_content&.empty?) }
           return unless node_children.empty?
 
           add_offense(begin_node) { |corrector| corrector.remove(begin_node) }
         end
+
+        private
+
+        def in_percent_w_upper_literal?(begin_node)
+          array_node = begin_node.each_ancestor(:array).first
+          return false unless array_node
+
+          array_node.source.start_with?('%W')
```

Comment:
Considering `%I` as well, wouldn’t `percent_literal?` be more appropriate?

```suggestion
          array_node.percent_literal?
```